h2o-r/tests/testdir_algos/deeplearning/runit_deeplearning_poisson_offset.R

```diff
@@ -57,10 +57,10 @@ test <- function() {
 	print(mean(ph[,1]))
 	print(min(ph[,1]))
 	print(max(ph[,1]))
-	expect_equal(-408156.9, mean_deviance, tolerance=1e-5)
-	expect_equal(20208.67, mean(ph[,1]), tolerance=1e-5 )
-	expect_equal(572.5512, min(ph[,1]), tolerance=1e-5 )
-	expect_equal(217348, max(ph[,1]), tolerance=1e-5 )
+	expect_equal(304397, mean_deviance, tolerance=1)
```

Comment:
am I crazy  or tolerance=1 isn't test anything? The value of mean_deviance is 4.049403 but 304397 passed 😕